pinot-core/src/main/java/org/apache/pinot/core/segment/virtualcolumn/VirtualColumnContext.java

```diff
@@ -18,33 +18,23 @@
  */
 package org.apache.pinot.core.segment.virtualcolumn;
 
+import org.apache.pinot.common.data.FieldSpec;
+
+
 /**
  * Context used to pass arguments to the virtual column provider.
  */
 public class VirtualColumnContext {
-  private String _hostname;
-  private String _segmentName;
-  private String _columnName;
+  private FieldSpec _fieldSpec;
```

Comment:
This is not related to your PR, but since you are touching it and have some context, some more comments on what VirtualColumnContext does will be useful. To me, the class seems to be a miscellaneous class of all things needed for a virtual column that cannot be determined in other places? If you can add a better class description that will be awesome, thank you.